### PR TITLE
[zh] sync blog: istio-graduates-within-cncf

### PR DESCRIPTION
**Pull request by windsonsea**
Need to wait for en #13544 

- [x] Docs

### CODE REVIEW

File: content/zh/blog/2023/istio-graduates-within-cncf/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+Sidecar 部署仍然是使用 Istio 的推荐方法，而我们的
+[1.19 版本](https://github.com/istio/istio/wiki/Istio-Release-1.19)
+将对 Kubernetes 1.28 中 Alpha 级别的
+[全新 Sidecar 容器特性](https://github.com/kubernetes/kubernetes/pull/116429)提供支持。
+
+我们欢迎微软加入 Istio 社区，
+微软[决定将 Open Service Mesh 项目归档并协作推进 Istio](https://openservicemesh.io/blog/osm-project-update/)。
+Istio 作为[活跃度排名第三的 CNCF 项目](https://all.devstats.cncf.io/d/53/projects-health-table?orgId=1)，
+得到了 [20 多家供应商的支持](/zh/about/ecosystem/)以及
+[数十家公司持续做贡献](https://istio.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=prs)，
```
Comment (wilsonwu) [inline content/zh/blog/2023/istio-graduates-within-cncf/index.md:26]:
```suggestion
[数十家公司的持续贡献](https://istio.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=prs)，
```

File: content/zh/blog/2023/istio-graduates-within-cncf/index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+CNCF 发布的公告以及
+[Intel](https://www.intel.com/content/www/us/en/developer/articles/community/Intel-Service-Mesh-Optimizes-and-Protects-Istio-Service-Mesh)、
+[Red Hat](https://cloud.redhat.com/blog/red-hat-congratulates-istio-on-graduating-at-the-cncf)、
+[Tetrate](https://tetrate.io/blog/istio-service-mesh-graduates-cncf/)
+和 [Solo.io](https://www.solo.io/blog/istio-graduates-cncf)（还有更多公司）
+发布的博文都展示了如今 Istio 项目参与者的许多想法和感受。
+
+我们也联系了一些以下已离开 Istio 项目的贡献者，让他们分享自己的想法。
+
+{{< /quote >}}
```
Comment (wilsonwu) [inline content/zh/blog/2023/istio-graduates-within-cncf/index.md:52]:
```suggestion
```

File: content/zh/blog/2023/istio-graduates-within-cncf/index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+{{< quote caption="Douglas Reid，最初的 Istio 工程师和 Steamship 创始工程师" >}}
+当我们为 Istio 构建初始原型时，我们希望其他人能够看到自己所创造的价值，
+并以积极的方式影响组织构建、管理和监控生产服务的方式。
+从 CNCF 毕业是对这些最初愿景的实现，这远远超出了任何合理的预期。
+当然，这样的里程碑只有在充满热情、知识渊博且专注的大量个人贡献下才能实现。
+这一成就源于多年来贡献者们的友好交流、耐心和所分享的专业知识。
+愿这个项目继续发展，并帮助用户在未来很多年内都能交付安全、可靠的服务！
+{{< /quote >}}
+
+{{< /quote >}}
```
Comment (hanxiaop) [inline content/zh/blog/2023/istio-graduates-within-cncf/index.md:91]: Is thit an extra quote?

File: content/zh/blog/2023/istio-graduates-within-cncf/index.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+开源协作项目的一个伟大之处在于无论贡献者受雇于哪家公司，他们与项目本身的关联不会中断。
+六年来，一些最初的贡献者基于 Istio 创立了新公司；
+一些人跳槽到了其他公司继续支持 Istio；
+还有一些人至今仍在 Google 或 IBM 从事 Istio 相关的工作。
+
+CNCF 发布的公告以及
+[Intel](https://www.intel.com/content/www/us/en/developer/articles/community/Intel-Service-Mesh-Optimizes-and-Protects-Istio-Service-Mesh)、
+[Red Hat](https://cloud.redhat.com/blog/red-hat-congratulates-istio-on-graduating-at-the-cncf)、
+[Tetrate](https://tetrate.io/blog/istio-service-mesh-graduates-cncf/)
+和 [Solo.io](https://www.solo.io/blog/istio-graduates-cncf)（还有更多公司）
```
Comment (wilsonwu) [inline content/zh/blog/2023/istio-graduates-within-cncf/index.md:49]:
```suggestion
[Tetrate](https://tetrate.io/blog/istio-service-mesh-graduates-cncf/)、
[Solo.io](https://www.solo.io/blog/istio-graduates-cncf)
和 [DaoCloud](https://blog.daocloud.io/8970.html)（还有更多公司）
```

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**